guide/07-working-with-bigdata/07-use_proximity.ipynb

```diff
@@ -18,11 +18,15 @@
    "cell_type": "markdown",
```

Comment:
<p>bracket is missing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1305/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>